ingest.py

```diff
@@ -5,24 +5,38 @@
 from langchain.embeddings import OpenAIEmbeddings
 from langchain.text_splitter import RecursiveCharacterTextSplitter
 from langchain.vectorstores.faiss import FAISS
-
+from langchain.document_loaders import RecursiveUrlLoader
+import numpy as np
 
 def ingest_docs():
     """Get documents from web pages."""
-    loader = ReadTheDocsLoader("langchain.readthedocs.io/en/latest/")
+    loader = ReadTheDocsLoader("api.python.langchain.com/en/latest", features="lxml")
```

Comment:
did you check that this is actually producing good results? the html content has changed a bit, may not be as relevant?

Reply:
switched to recursiveurlloader, which seems like it produces better